packages/slate-react/src/components/editable.tsx

```diff
@@ -624,19 +624,52 @@ export const Editable = (props: EditableProps) => {
               ) {
                 const node = ReactEditor.toSlateNode(editor, event.target)
                 const path = ReactEditor.findPath(editor, node)
-                const start = Editor.start(editor, path)
-                const end = Editor.end(editor, path)
+                // Check if user clicks thrice
+                if (event.detail === 3) {
+                  // Check the closest element block above the selection
+                  const elem = Editor.above(editor, {
+                    at: path,
+                    match: n => Editor.isBlock(editor, n),
+                  })
+                  if (elem) {
+                    // Get all the children of the element above
+                    const children = Node.children(editor, elem[1])
+                    const childrenPath = []
+                    // Loop through the generator of element children and create a list of their respective paths.
```

Comment:
Please run the linter on this file, this comment and a few below likely exceed the line length rules.